install-and-configure/install/provider-installations/openshift-kubecost-install.md

```diff
@@ -60,6 +60,41 @@ If you have not opted to do so during installation, it may be necessary to creat
 
 After installation, wait for all pods to be ready. Kubecost will begin collecting data and may take up to 15 minutes for the UI to reflect the resources in the local cluster.
 
+### Using in-cluster Prometheus
+
+Kubecost can now be installed without the bundled Prometheus and using the in-cluster Prometheus of OpenShift. This method requires some additional setup steps but allows for better integration with existing OpenShift monitoring infrastructure without the need to create an additional prometheus instance.
+
+1. First, add the following label to the namespace where Kubecost will be deployed:
+
+```sh
+oc label namespace kubecost openshift.io/cluster-monitoring=true
+```
+
+2. Install Kubecost with the following command:
+
+```sh
+helm upgrade --install kubecost kubecost/cost-analyzer -n kubecost --create-namespace \
+-f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-openshift.yaml \
```

Comment:
We need to not tell users to fetch values from the `develop` branch as this will inevitably result in a mismatch at some point and an unsupported configuration. I suggest either a placeholder value or an export to capture the release corresponding to branch name. I know we do this elsewhere in the docs and it's on my to-do list to fix.

Reply:
understood. Makes sense. I should go ahead with adding a placeholder in that case. I see we are doing that in some places